File: kotlin/1092.B.kt
fun main(){
    val n=readLine()!!.toInt()
    var s=readLine()!!.split(" ").map(String::toInt)
    var d=0
    s=s.toList().sorted()
    var k= s.slice(0..s.size-1 step 2).zip(s.slice(1..s.size-1 step 2))
    for(i in k)
                d+=Math.abs(i.first-i.second)
    
    print(d)
}

File: dotNet/1092.B.cs
using System;
using System.Linq;

class Program
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int[] s = Console.ReadLine().Split().Select(int.Parse).ToArray();

        Array.Sort(s);
        int sum = 0;
        for (int i = 0; i < n / 2; i++)
        {
            sum += Math.Abs(s[2 * i] - s[2 * i + 1]);
        }
        
        Console.WriteLine(sum);
    }
}
